emails/passwordless-login-code.html

```diff
@@ -1,189 +1,43 @@
-            {% if operation == 'change_email' %}
-
-            <p style="font-size: 1.2em;line-height: 1.3;-webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%;">Your
-              email address has been updated.</p>
-
-            {% else %}
-
-            <!-- Signup email content -->
-            {% if send == 'link' or send == 'link_ios' or send == 'link_android' %}
-
-            <p style="font-size: 1.2em;line-height: 1.3;-webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%;">
-              Click and confirm that you want to sign in to {{ application.name }}. This link will expire in five
-              minutes.</p>
-
-            <div style="text-align:center">
-              <a id="signIn"
-                style="text-transform: uppercase;letter-spacing: 1px;color: #ffffff;text-decoration: none;display: inline-block;min-height: 48px;line-height: 48px;padding-top: 0;padding-right: 26px;padding-bottom: 0;margin: 20px 0;padding-left: 26px;border: 0;outline: 0;background: #006400;font-size: 14px;font-style: normal;font-weight: 400;text-align: center;white-space: nowrap;border-radius: 3px;text-overflow: ellipsis;max-width: 280px;overflow: hidden;-webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%;"
-                href="{{ link }}">Sign in to {{ application.name }}</a>
-            </div>
-
-            <p style="-webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%;">Or sign in using this link:</p>
-            <p style="-webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%;"><a
-                style="font-size: 12px; color: #A9B3BC; text-decoration: none;word-break: break-all;-webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%;"
-                href="{{ link }}">{{ link }}</a></p>
-
-            {% elsif send == 'code' %}
-
-            <p style="font-size: 1.4em; line-height: 1.3;">Your sign in code is: <b>{{ code }}</b></p>
-            <br>
-            <p style="-webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%;">This code is valid for the next <b>15
-                minutes</b> only.</p>
-            {% endif %}
-
-            {% endif %}
-
-            <p style="-webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%;">If you need technical support, here
-              are some <a href="https://support.freecodecamp.org/">frequently asked questions and answers</a>.</p>
-
-            <br>
-            Happy Coding.
-            <br>
-
-            <strong>- the {{ application.name }} team</strong>
-
-          </div>
-        </td>
-      </tr>
-    </table>
-  </center>
-</body>
-
-</html>
\ No newline at end of file
+<!-- Email change content -->
+{% if operation == 'change_email' %}
+<p>
+  Your email address has been updated.
+</p>
+{% else %}
+
+<!-- Signup email content -->
+{% if send == 'link' or send == 'link_ios' or send == 'link_android' %}
+<p>
+  Click and confirm that you want to sign in to {{ application.name }}. This link will expire in <b>15
```

Comment:
```suggestion
  Click to confirm that you want to sign in to {{ application.name }}. This link will expire in <b>15
```
Nit. Also, should 'This link will expire..' be below both?  Then it seems natural to say 'These links will expire...'.